CI lint broken on main: unchecked resp.Body.Close in internal/github/client.go

## Summary

Commit `848fe202` ("feat: add gate bead instruction template and GitHub API client", fix-merge PR #3311) introduced two unchecked `resp.Body.Close` calls in the new `internal/github/client.go`, breaking the `golangci-lint` CI check on main for all PRs.

## Error

```
internal/github/client.go:101:23: Error return value of `resp.Body.Close` is not checked (errcheck)
internal/github/client.go:147:23: Error return value of `resp.Body.Close` is not checked (errcheck)
```

## Impact

Every PR targeting main currently fails the Lint CI check regardless of its own changes.

--- a/internal/github/client.go
+++ b/internal/github/client.go
@@ -98,7 +98,7 @@ func (c *Client) restRequest(ctx context.Context, method, path string, body any,
 	if err != nil {
 		return fmt.Errorf("github: %s %s: %w", method, path, err)
 	}
-	defer resp.Body.Close()
+	defer func() { _ = resp.Body.Close() }()
 
 	respBody, err := io.ReadAll(resp.Body)
 	if err != nil {
@@ -144,7 +144,7 @@ func (c *Client) graphqlRequest(ctx context.Context, query string, variables map
 	if err != nil {
 		return fmt.Errorf("github: graphql: %w", err)
 	}
-	defer resp.Body.Close()
+	defer func() { _ = resp.Body.Close() }()
 
 	respBody, err := io.ReadAll(resp.Body)
 	if err != nil {

